### PR TITLE
Conform to doc requirements so that the docs look more beautiful

### PR DESCRIPTION
**Pull request by alexjbest**
These are mostly either of the form of unindenting text that was accidentally put in a code block, or correctly adding `::` or `EXAMPLES::` so that docs render nicely.

### 📝 Checklist

- [ ] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [ ] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/coding/guruswami_sudan/interpolation.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def _interpolation_matrix_problem(points, tau, parameters, wy):
     EXAMPLES:
 
     The following parameters arise from Guruswami-Sudan decoding of an [6,2,5]
-    GRS code over F(11) with multiplicity 2 and list size 4.
+    GRS code over F(11) with multiplicity 2 and list size 4.::
```
Comment (tscrim) [inline src/sage/coding/guruswami_sudan/interpolation.py]:
```suggestion
    GRS code over F(11) with multiplicity 2 and list size 4. ::
```
or
```suggestion
    GRS code over F(11) with multiplicity 2 and list size 4::
```
Having no space means the result is `.:` in the compiled doc. A space before the `::` means the colon is not displayed.

File: src/sage/combinat/subset.py [hunk 2]
```diff
@@ -342,9 +342,9 @@ def cardinality(self):
             sage: Subsets(3).cardinality()
             8
 
-        TESTS::
+        TESTS:
 
-            ``__len__`` should return a Python int.
+        ``__len__`` should return a Python int.::
```
Comment (tscrim) [inline src/sage/combinat/subset.py]: Same comment as above: either `int. ::` or `int::` (I would opt for the latter here).

File: src/sage/combinat/tableau.py [hunk 3]
```diff
@@ -233,7 +233,7 @@ def __init__(self, parent, t, check=True):
 
         A tableau is shallowly immutable. See :trac:`15862`. The entries
         themselves may be mutable objects, though in that case the
-        resulting Tableau should be unhashable.
+        resulting Tableau should be unhashable.::
```
Comment (tscrim) [inline src/sage/combinat/tableau.py]: Same here.

File: src/sage/geometry/newton_polygon.py [hunk 4]
```diff
@@ -630,13 +630,15 @@ def __init__(self):
...
+        EXAMPLES::
+
             sage: from sage.geometry.newton_polygon import ParentNewtonPolygon
             sage: ParentNewtonPolygon()
             Parent for Newton polygons
 
         TESTS:
 
-        This class is a singleton.
+        This class is a singleton.::
```
Comment (tscrim) [inline src/sage/geometry/newton_polygon.py]: Same here.

File: src/sage/modules/fg_pid/fgp_element.py [hunk 5]
```diff
@@ -156,7 +156,7 @@ def _add_(self, other):
             sage: 0 + x
             (1, 0)
 
-        We test canonical coercion from V and W.
+        We test canonical coercion from V and W.::
```
Comment (tscrim) [inline src/sage/modules/fg_pid/fgp_element.py]: Same here.

File: src/sage/modules/torsion_quadratic_module.py [hunk 6]
```diff
@@ -850,7 +850,7 @@ def orthogonal_group(self, gens=None, check=False):
             [1 0 0]
             [0 0 1]
 
-        We compute the kernel of the action of the orthogonal group of `L` on the discriminant group.
+        We compute the kernel of the action of the orthogonal group of `L` on the discriminant group.::
```
Comment (tscrim) [inline src/sage/modules/torsion_quadratic_module.py]: Same here.

File: src/sage/plot/plot3d/plot3d.py [hunk 7]
```diff
@@ -248,7 +248,7 @@ def __init__(self, dep_var, indep_vars):
 
         Because the base :class:`_Coordinates` class automatically checks the
         initializing variables with the transform method, :class:`_Coordinates`
-        cannot be instantiated by itself.  We test a subclass.
+        cannot be instantiated by itself.  We test a subclass.::
```
Comment (tscrim) [inline src/sage/plot/plot3d/plot3d.py]: Same here.

File: src/sage/quivers/morphism.py [hunk 8]
```diff
@@ -1108,7 +1108,7 @@ def algebraic_dual(self):
             Representation with dimension vector (5, 2, 1, 1, 4)
 
         The algebraic dual of an indecomposable projective is the indecomposable
-        projective of the same vertex in the opposite quiver.
+        projective of the same vertex in the opposite quiver.::
```
Comment (tscrim) [inline src/sage/quivers/morphism.py]: Same here.

File: src/sage/rings/padics/lattice_precision.py [hunk 9]
```diff
@@ -974,7 +974,7 @@ def precision_lattice(self, elements=None):
             [   0 2048]
 
         If the precision module does not project to a lattice,
-        an error is raised.
+        an error is raised.::
```
Comment (tscrim) [inline src/sage/rings/padics/lattice_precision.py]: Same here and the other change to this file.

File: src/sage/rings/padics/padic_base_leaves.py [hunk 10]
```diff
@@ -904,7 +904,7 @@ def _coerce_map_from_(self, R):
             True
 
         Note that coerce map does not exist between ``p``-adic rings with
-        lattice precision and other ``p``-adic rings.
+        lattice precision and other ``p``-adic rings.::
```
Comment (tscrim) [inline src/sage/rings/padics/padic_base_leaves.py]: Same for this file.

File: src/sage/rings/polynomial/infinite_polynomial_ring.py [hunk 11]
```diff
@@ -646,7 +646,7 @@ class InfinitePolynomialRing_sparse(CommutativeRing):
         sage: Z = InfinitePolynomialRing_sparse(QQ, ['x','y'], 'lex')
 
     Nevertheless, since infinite polynomial rings are supposed to be unique
-    parent structures, they do not evaluate equal.
+    parent structures, they do not evaluate equal.::
```
Comment (tscrim) [inline src/sage/rings/polynomial/infinite_polynomial_ring.py]: Here too.

File: src/sage/rings/padics/padic_valuation.py [hunk 12]
```diff
@@ -439,7 +439,7 @@ class pAdicValuation_base(DiscreteValuation):
         sage: QQ.valuation(5)
         5-adic valuation
 
-     For `p`-adic rings, ``p`` has to match the `p` of the ring.
+     For `p`-adic rings, ``p`` has to match the `p` of the ring.::
```
Comment (tscrim) [inline src/sage/rings/padics/padic_valuation.py]: And here.

File: src/sage/rings/padics/padic_lattice_element.py [hunk 13]
```diff
@@ -1010,7 +1010,7 @@ def unit_part(self):
             sage: b.unit_part()
             1 + 16*17 + O(17^3)
 
-        If the element is indistinguishable from zero, an error is raised.
+        If the element is indistinguishable from zero, an error is raised.::
```
Comment (tscrim) [inline src/sage/rings/padics/padic_lattice_element.py]: Same here.

File: src/sage/rings/padics/padic_extension_generic.py [hunk 14]
```diff
@@ -510,7 +510,7 @@ def construction(self, forbid_frac_field=False):
             sage: c(R0) == R
             True
 
-        For a field, by default we return a fraction field functor.
+        For a field, by default we return a fraction field functor.::
```
Comment (tscrim) [inline src/sage/rings/padics/padic_extension_generic.py]: Same here.

File: src/sage/rings/polynomial/ore_function_element.py [hunk 15]
```diff
@@ -845,7 +845,7 @@ def reduced_trace(self, var=None):
             3/(z^2 + 2)
 
         The reduced trace lies in the center of `S`, which is the fraction field
-        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`.
+        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`.::
```
Comment (tscrim) [inline src/sage/rings/polynomial/ore_function_element.py]: Also here.

File: src/sage/sets/finite_set_maps.py [hunk 16]
```diff
@@ -305,7 +305,7 @@ def an_element(self):
 
         An exception :class:`~sage.categories.sets_cat.EmptySetError`
         is raised if this set is empty, that is if the codomain is
-        empty and the domain is not.
+        empty and the domain is not.::
```
Comment (tscrim) [inline src/sage/sets/finite_set_maps.py]: Same here.

File: src/sage/sets/set.py [hunk 17]
```diff
@@ -616,7 +616,7 @@ def __contains__(self, x):
             False
 
         Finite fields better illustrate the difference between
-        ``__contains__`` for objects and their underlying sets.
+        ``__contains__`` for objects and their underlying sets.::
```
Comment (tscrim) [inline src/sage/sets/set.py]: Also here.

**Review by tscrim — CHANGES_REQUESTED** [review]
All are just a minor formatting issue of `.::` displaying at `.:` in the doc. Either change to `. ::` (for `.` output) or `::` (for `:` output).

File: src/sage/combinat/subset.py (hunk 2 above)
Comment (alexjbest) [reply src/sage/combinat/subset.py]:
```suggestion
        ``__len__`` should return a Python int::
```

File: src/sage/combinat/tableau.py (hunk 3 above)
Comment (alexjbest) [reply src/sage/combinat/tableau.py]:
```suggestion
        resulting Tableau should be unhashable. ::
```

File: src/sage/geometry/newton_polygon.py (hunk 4 above)
Comment (alexjbest) [reply src/sage/geometry/newton_polygon.py]:
```suggestion
        This class is a singleton::
```

File: src/sage/modules/fg_pid/fgp_element.py (hunk 5 above)
Comment (alexjbest) [reply src/sage/modules/fg_pid/fgp_element.py]:
```suggestion
        We test canonical coercion from V and W::
```

File: src/sage/modules/torsion_quadratic_module.py (hunk 6 above)
Comment (alexjbest) [reply src/sage/modules/torsion_quadratic_module.py]:
```suggestion
        We compute the kernel of the action of the orthogonal group of `L` on the discriminant group::
```

File: src/sage/plot/plot3d/plot3d.py (hunk 7 above)
Comment (alexjbest) [reply src/sage/plot/plot3d/plot3d.py]:
```suggestion
        cannot be instantiated by itself.  We test a subclass::
```

File: src/sage/quivers/morphism.py (hunk 8 above)
Comment (alexjbest) [reply src/sage/quivers/morphism.py]:
```suggestion
        projective of the same vertex in the opposite quiver. ::
```

File: src/sage/rings/padics/lattice_precision.py (hunk 9 above)
Comment (alexjbest) [reply src/sage/rings/padics/lattice_precision.py]:
```suggestion
        an error is raised. ::
```

File: src/sage/rings/padics/padic_base_leaves.py (hunk 10 above)
Comment (alexjbest) [reply src/sage/rings/padics/padic_base_leaves.py]:
```suggestion
        lattice precision and other ``p``-adic rings. ::
```

File: src/sage/rings/polynomial/infinite_polynomial_ring.py (hunk 11 above)
Comment (alexjbest) [reply src/sage/rings/polynomial/infinite_polynomial_ring.py]:
```suggestion
    parent structures, they do not evaluate equal. ::
```

File: src/sage/rings/padics/padic_valuation.py (hunk 12 above)
Comment (alexjbest) [reply src/sage/rings/padics/padic_valuation.py]:
```suggestion
     For `p`-adic rings, ``p`` has to match the `p` of the ring. ::
```

File: src/sage/rings/padics/padic_lattice_element.py (hunk 13 above)
Comment (alexjbest) [reply src/sage/rings/padics/padic_lattice_element.py]:
```suggestion
        If the element is indistinguishable from zero, an error is raised::
```

File: src/sage/rings/padics/padic_extension_generic.py (hunk 14 above)
Comment (alexjbest) [reply src/sage/rings/padics/padic_extension_generic.py]:
```suggestion
        For a field, by default we return a fraction field functor. ::
```

File: src/sage/rings/polynomial/ore_function_element.py (hunk 15 above)
Comment (alexjbest) [reply src/sage/rings/polynomial/ore_function_element.py]:
```suggestion
        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`::
```

File: src/sage/sets/finite_set_maps.py (hunk 16 above)
Comment (alexjbest) [reply src/sage/sets/finite_set_maps.py]:
```suggestion
        empty and the domain is not. ::
```

File: src/sage/sets/set.py (hunk 17 above)
Comment (alexjbest) [reply src/sage/sets/set.py]:
```suggestion
        ``__contains__`` for objects and their underlying sets::
```

File: src/sage/rings/padics/padic_base_leaves.py [hunk 18]
```diff
@@ -913,7 +913,7 @@ def _coerce_map_from_(self, R):
             False
 
         Similarly there is no coercion maps between ``p``-adic rings with
-        different labels.
+        different labels.::
```
Comment (alexjbest) [inline src/sage/rings/padics/padic_base_leaves.py]:
```suggestion
        different labels. ::
```

File: src/sage/rings/padics/padic_base_leaves.py [hunk 19]
```diff
@@ -1033,7 +1033,7 @@ def _coerce_map_from_(self, R):
             True
 
         Note that coerce map does not exist between ``p``-adic fields with
-        lattice precision and other ``p``-adic rings.
+        lattice precision and other ``p``-adic rings.::
```
Comment (alexjbest) [inline src/sage/rings/padics/padic_base_leaves.py]:
```suggestion
        lattice precision and other ``p``-adic rings. ::
```

File: src/sage/rings/polynomial/ore_function_element.py [hunk 20]
```diff
@@ -906,7 +906,7 @@ def reduced_norm(self, var=None):
             (z + 2)/(z^2 + 4)
 
         The reduced norm lies in the center of `S`, which is the fraction field
-        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`.
+        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`.::
```
Comment (alexjbest) [inline src/sage/rings/polynomial/ore_function_element.py]:
```suggestion
        of a univariate polynomial ring in the variable `z = x^3` over `GF(5)`. ::
```

File: src/sage/rings/polynomial/term_order.py [hunk 21]
```diff
@@ -1790,7 +1790,7 @@ def singular_moreblocks(self):
         TESTS:
 
         The 'degneglex' ordering is somehow special: SINGULAR handles it
-        using an extra weight vector block.
+        using an extra weight vector block.::
```
Comment (alexjbest) [inline src/sage/rings/polynomial/term_order.py]:
```suggestion
        using an extra weight vector block. ::
```

File: src/sage/rings/padics/lattice_precision.py [hunk 22]
```diff
@@ -2689,7 +2689,7 @@ def precision_lattice(self, elements=None):
...
```
Comment (alexjbest) [inline src/sage/rings/padics/lattice_precision.py]:
```suggestion
        an error is raised. ::
```

**Review by fchapoton — APPROVED** [review]
looks good, thanks

**Pull request merged**